### ISSUE TITLE
Unable to add a Turtle triple to the store

### ISSUE THREAD

**Issue by wouterbeek** [oxigraph/oxigraph#200]
### Observation

When I try to add a triple to the server, I observe the following error:

```
thread '<unnamed>' panicked at 'attempt to divide by zero', /home/wouter/.cargo/registry/src/github.com-1ecc6299db9ec823/oxigraph_server-0.3.1/src/main.rs:87:33
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Any { .. }', /home/wouter/.cargo/registry/src/github.com-1ecc6299db9ec823/oxigraph_server-0.3.1/src/main.rs:110:31
```

I use the server's load option, as follows:

```sh
oxigraph_server --location ${some-dir} load --file test.ttl
```

I have tested this with different Turtle files `test.ttl`, but they all emit the same error.

### Expected

The server to be able to load RDF from a Turtle file.

### Steps to reproduce

1. Store the following Turtle snippet in a file called `test.ttl`:

   ```ttl
   base <https://example.com/>
   <s><p><o>.
   ```
3. Run the following command: `oxigraph_server --location ${some-dir} load --file test.ttl`
4. Observe the above shared error message

### Environment

I'm on the last Ubuntu LTS version.  I can share other environment properties if needed.

**Comment by Tpt** [comment]
Hi! Thank you for trying Oxigraph. Sorry for this bug. It seems I forgot to test the printing output of the loader of very small files that loads in less than a second. Hence the bug. The automated tests have the logging disabled so it slipped through...

I will release a new "patch release" soon to fix that.

**Comment by Tpt** [comment]
I have released v0.3.2 that should fix the problem.

**Comment by wouterbeek** [comment]
Thanks!
